frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -64,6 +124,12 @@
                 default: {}
             }
         },
+        mixins: [
+            // call groups plugin input mixin for drawer
+            countlyVue.container.tabsMixin({
+                "groupsInput": "groups/input"
```

Comment:
looks like you are using tabsMixin to add tabs.
But i could not find any reference to the `groupsInput` or to the `cly-dynamic-tabs` in the pr?
Was this on purpose or mistake ?

Reply:
:+1: